pandas/plotting/_core.py

```diff
@@ -1935,52 +1935,134 @@ def plot_series(data, kind='line', ax=None,                    # Series unique
 
 
 _shared_docs['boxplot'] = """
-    Make a box plot from DataFrame column optionally grouped by some columns or
-    other inputs
+    Make a box-and-whisker plot from DataFrame column optionally grouped
+    by some columns or other inputs. The box extends from the Q1 to Q3
+    quartile values of the data, with a line at the median (Q2).
+    The whiskers extend from the edges of box to show the range of the data.
+    Flier points (outliers) are those past the end of the whiskers.
+    The position of the whiskers is set by default to 1.5 IQR (`whis=1.5``)
+    from the edge of the box.
+
+    For further details see
+    Wikipedia's entry for `boxplot <https://en.wikipedia.org/wiki/Box_plot/>`_.
 
     Parameters
     ----------
-    data : the pandas object holding the data
     column : column name or list of names, or vector
```

Comment:
The [Parameter types](https://python-sprints.github.io/pandas/guide/pandas_docstring.html#parameter-types) do not follow the docstring guide. 

For example, I suggest that `column` should be something like:
```
column : str or list of str, optional
    Column name or list of names, or vector. Can be any valid input to groupby.
```

An it applies to the other params as well.

Reply:
Thanks for the suggestion!